languages/cpp/exercises/concept/numbers/numbers_test.cpp

```diff
@@ -0,0 +1,68 @@
+#include "numbers.h"
+
+#include "test/catch.hpp"
+
+TEST_CASE("Production rate per hour for speed zero") {
+    auto rate = assembly_line::production_rate_per_hour(0);
+    static_assert(std::is_same<double, decltype(rate)>::value,
+                  "Returned value should be of type double");
+    REQUIRE(0.0 == rate);
+}
+
+#if defined(EXERCISM_RUN_ALL_TESTS)
+TEST_CASE("Production rate_per_hour_for_speed_one") {
+    auto rate = assembly_line::production_rate_per_hour(1);
+    static_assert(std::is_same<double, decltype(rate)>::value,
+                  "Returned value should be of type double");
+    REQUIRE(221.0 == rate);
```

Comment:
Should we really compare exact values? I'd suggest
REQUIRE(Approx(221.0).margin(0.001) == rate);

Reply:
That is a very good point! Indeed, exact comparison of floats is generally bad idea. I'll add a margin.